geopandas/io/tests/test_arrow.py

```diff
@@ -357,98 +387,103 @@ def test_parquet_invalid_metadata(tmpdir, geo_meta, error):
         read_parquet(filename)
 
 
-def test_parquet_subset_columns(tmpdir):
+def test_subset_columns(tmpdir, file_format):
     """Reading a subset of columns should correctly decode selected geometry
     columns.
     """
+    reader, writer = file_format
 
     test_dataset = "naturalearth_lowres"
     df = read_file(get_path(test_dataset))
 
     filename = os.path.join(str(tmpdir), "test.pq")
-    df.to_parquet(filename)
-    pq_df = read_parquet(filename, columns=["name", "geometry"])
+    writer(df, filename)
+    pq_df = reader(filename, columns=["name", "geometry"])
 
     assert_geodataframe_equal(df[["name", "geometry"]], pq_df)
 
     with pytest.raises(
         ValueError, match="No geometry columns are included in the columns read"
     ):
-        read_parquet(filename, columns=[])
+        reader(filename, columns=["name"])
```

Comment:
The Feather reader apparently doesn't work with an empty list for columns selection (it then returns all columns), so changing the test here to only read a non-geometry column